SECURITY.md

```diff
@@ -0,0 +1,41 @@
+<!-- BEGIN MICROSOFT SECURITY.MD V0.0.5 BLOCK -->
+
+## Security
+
+Microsoft takes the security of our software products and services seriously, which includes all source code repositories managed through our GitHub organizations, which include [Microsoft](https://github.com/Microsoft), [Azure](https://github.com/Azure), [DotNet](https://github.com/dotnet), [AspNet](https://github.com/aspnet), [Xamarin](https://github.com/xamarin), and [our GitHub organizations](https://opensource.microsoft.com/).
+
+If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/en-us/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
```

Comment:
```suggestion
If you believe you have found a security vulnerability in any Microsoft-owned repository that meets [Microsoft's definition of a security vulnerability](https://docs.microsoft.com/previous-versions/tn-archive/cc751383(v=technet.10)), please report it to us as described below.
```

Reply:
Can you remove the "en-us" from the links to make the links language neutral?

Reply:
I believe you can suggest changes at https://github.com/microsoft/repo-templates/blob/main/shared/SECURITY.md to modify the global template. @jeffwilcox, FYI

Reply:
Yes, appreciate any such improvements to the source file (and also incrementing the "version" in the comment), we've taken the content from security teams in the past.

Reply:
I don't think we should link to a page which says "We're no longer updating this content regularly. Check the [Microsoft Product Lifecycle](https://docs.microsoft.com/en-us/lifecycle/products) for information about how this product, service, technology, or API is supported." at the top.

Reply:
Sure does seem like the links should be aka.ms links that could be more resilient. I'm trying to track down who's still at the company who helped roll out the initial SECURITY.MD template, since updates are expensive and hard to do.